app/assets/javascripts/dialog_field_refresh.js

```diff
@@ -23,83 +23,59 @@ var dialogFieldRefresh = {
   refreshCheckbox: function(fieldName, fieldId) {
     miqSparkleOn();
 
-    $.post('dynamic_checkbox_refresh', {name: fieldName}, function(data) {
-      $('.dynamic-checkbox-' + fieldId).prop('checked', data.values.checked);
-      dialogFieldRefresh.setReadOnly($('.dynamic-checkbox-' + fieldId), data.values.read_only);
-      miqSparkle(false);
```

Comment:
I was going to ask why we didn't need `miqSparkle(false);` anymore - but found it is now handled with `miqJqueryRequest`